tests/test_module.py

```diff
@@ -3,16 +3,20 @@
 Use 'pytest tests/test_module.py::test_name' to run a single test.
```

Comment:
The `test_module` name is just default - it should really be split up into `test_clue_analysis`, `test_eval_comparisons`, `test_evaluations`, and so on. All tests concerning things in module X should then be put in test module "test_X"